source/reference/json/cloud-backup-schedule-config-file.txt

```diff
@@ -0,0 +1,241 @@
+
+   * - ``policies.``
+       ``policyItems.``
+       ``frequencyInterval``
+     - integer
+     - Number that indicates the frequency interval for a set of
+       snapshots. A value of ``1`` specifies the first instance of the
+       corresponding ``frequencyType``.
+
+       - In a monthly policy item, ``1`` indicates that the monthly
+         snapshot occurs on the first day of the month and ``40``
+         indicates the last day of the month.
+
+       - In a weekly policy item, ``1`` indicates that the weekly
+         snapshot occurs on Monday and ``7`` indicates Sunday.
+
+       - In an hourly policy item, you can set the frequency interval to
+         ``1``, ``2``, ``4``, ``6``, ``8``, or ``12``. For hourly policy
+         items for NVMe clusters, MongoDB Cloud only accepts ``12`` as
+         the frequency interval value.
+
+   * - ``policies.``
+       ``policyItems.``
+       ``frequencyType``
+     - string
+     - Human-readable label that identifies the frequency type
+       associated with the backup policy.
+
+   * - ``policies.``
+       ``policyItems.``
+       ``retentionUnit``
+     - string
+     - Unit of time in which MongoDB Cloud measures snapshot retention.
+
+   * - ``policies.``
+       ``policyItems.``
+       ``retentionValue``
+     - integer
+     - Duration in days, weeks, or months that MongoDB Cloud retains the
+       snapshot. For less frequent policy items, MongoDB Cloud requires
+       that you specify a value greater than or equal to the value
+       specified for more frequent policy items. 
+
+       For example: If the hourly policy item specifies a retention of two
+       days, you must specify two days or greater for the retention of the
+       weekly policy item. 
+
+   * - ``referenceHourOfDay``
+     - integer
+     - Hour of day in Coordinated Universal Time (UTC) that represents
+       when MongoDB Cloud takes the snapshot.
+
+   * - ``referenceMinuteOfHour``
+     - integer
+     - Minute of the ``referenceHourOfDay`` that represents when MongoDB
+       Cloud takes the snapshot.
+
+   * - ``restoreWindowDays``
+     - integer
+     - Number of previous days that you can restore back to with
+       Continuous Cloud Backup accuracy. You must specify a positive,
+       non-zero integer. This parameter applies to continuous cloud
+       backups only.
+
+   * - ``updateSnapshots``
+     - boolean
+     - Flag that indicates whether to apply the retention changes in the
+       updated backup policy to snapshots that MongoDB Cloud took
+       previously.
+
+   * - ``useOrgAndGroupNamesInExportPrefix``
+     - boolean
+     - Flag that indicates whether to use organization and project names
+       instead of organization and project UUIDs in the path to the
+       metadata files that MongoDB Cloud uploads to your |aws| bucket. 
+
+For a full list of available settings, see the API documentation for
+:oas-atlas-op:`Update Cloud Backup Schedule for One Cluster </cloudProvider>`.
```

Comment:
Question/nit: Is what's provided here not the full list? I suppose its safer to say this if they update the API and we forget to change it here, but something about this wording bugs me.

Reply:
You're right, I forgot to remove this (I was previously targeting to only put whatever was flagged as most used params, but ended up putting them all).